style.css

```diff
@@ -264,16 +270,230 @@ body {
   padding-bottom: 15px;
   padding-top: 15px;
   border-radius: 10px;
-  margin-right: 20px;
+  gap: 10px;
   display: flex;
 }
 
-.m-l {
-  margin-left: 20px;
-}
-
 .language {
   display: flex;
   justify-content: space-between;
   padding-right: 60px;
 }
+
+@media only screen and (min-width: 768px) {
+  .section-a-container {
+    background-image: url("pimages/header-background-mobile.png");
+    background-position: center;
+    background-size: 100% 90%;
+    background-repeat: no-repeat;
+    display: flex;
+    padding: 0 14em;
+    justify-content: center;
+    align-items: start;
+    flex-direction: column;
+    text-align: start;
+  }
```

Comment:
Please resize the background image on your headline section to match the figma design better. Currently, the image is stretched.

Your design:
<img width="507" alt="image" src="https://github.com/Ejikeokegbe/Mobile-First-portifolio/assets/37116322/d82227b3-a4cd-4f64-82eb-cddbe4ba75ae">

Figma design:
<img width="542" alt="image" src="https://github.com/Ejikeokegbe/Mobile-First-portifolio/assets/37116322/81f8d860-8981-437a-99d3-8709c56a9033">